### PR TITLE
Added GpuContextTest.

### PR DESCRIPTION
**Pull request by emilwestergren**
Main purpose is to test shared contexts.

@sebastianbaginski peer review

### CODE REVIEW

**Comment by sebastianbaginski** [comment]
Put a GPU flag around this test and I'm happy.

**Comment by sebastianbaginski** [comment]
... do we have to in test sources ?

**Comment by sebastianbaginski** [comment]
Looks like I was wrong, forget my comments. PR done.

File: test/draw/gpu/GpuContextTest.ooc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+use geometry
```
Comment (marcusnaslund) [inline test/draw/gpu/GpuContextTest.ooc]: Missing MIT license

**Comment by emilwestergren** [comment]
Updated.

**Comment by marcusnaslund** [comment]
Thank you. :)

**Pull request merged**